content/en/core/releases/4.5.0.md

```diff
@@ -0,0 +1,100 @@
+
+---
+title: "4.5.0 release notes"
+linkTitle: "4.5.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.5.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+- [#8513](https://github.com/medic/cht-core/issues/8513): Align "Target" components with Material design guidelines
+- [#8541](https://github.com/medic/cht-core/issues/8541): Options with long names look wrong in enketo selects
+- [#8686](https://github.com/medic/cht-core/issues/8686): Fix target UI to match Figma
```

Comment:
Remove this one from here. It's basically the same as the first and I think it'd be confusing if people try to work out what's different.

```suggestion
```